src/ontology/cl-edit.owl

```diff
@@ -4668,13 +4667,6 @@ AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000108 "WBbt:0006840")
 AnnotationAssertion(rdfs:label obo:CL_0000108 "cholinergic neuron")
 EquivalentClasses(obo:CL_0000108 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0014055)))
 
-# Class: obo:CL_0000109 (adrenergic neuron)
```

Comment:
Is that term removed because it “should be a synonym of noradrenergic neuron” (as stated in [this comment](https://github.com/obophenotype/cell-ontology/issues/2342#issuecomment-2639942670))?

Terms should never be entirely _removed_ like that. They should be _obsoleted_ instead – kept around, but clearly marked as obsolete with a `owl:deprecated` annotation.